src/app/components/url-box/url-box.component.html

```diff
@@ -104,9 +104,12 @@
     [ngClass]="{ 'url-box__button--disabled': isSubscribed || isLoading }"
     track-id="set_request"
     *ngIf="queryOperations.length < 2"
+    style="line-height: normal;"
```

Comment:
I think you want to set this here:

https://github.com/imolorhe/altair/blob/staging/src/scss/components/_url-box.scss#L69

Reply:
check